packages/network/src/workers/debug.ts

```diff
@@ -0,0 +1,3 @@
+import { debug as parentDebug } from "../debug";
+
+export const debug = parentDebug.extend("workers");
```

Comment:
this is fairly granular and maybe a little overkill but I figured it'd be a better starting place to expand on